iOS/Sources/ListFood/ListFoodPresenterViewController.swift

```diff
@@ -29,9 +29,9 @@ final class ListFoodPresenterViewController: ListFoodPresenter {
 }
 
 private extension ListFoodErrorViewModel {
-    private static var internalError = ListFoodErrorViewModel(title: localize("internal_error"), message: localize("this_should_not_happen"))
-    private static var invalidMonth = ListFoodErrorViewModel(title: localize("invalid_month"), message: localize("select_another_month"))
-    private static var notFound = ListFoodErrorViewModel(title: localize("source_not_found"), message: localize("this_should_not_happen"))
+    private static var internalError = ListFoodErrorViewModel(title: localize("Internal error"), message: localize("This should not happen"))
```

Comment:
__[SwiftLint]__ Line should be 140 characters or less: currently 141 characters ([link](https://sider.review/gh/repos/123621593/pulls/32#issue-4771807))

<sub>posted by Sider</sub>